config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
   resources :boards, only: [:index, :new, :create, :edit, :update, :destroy]
+  get '/login' => 'sessions#new'
+  post '/login' => 'sessions#create'
+  get '/logout' => 'sessions#destroy'
```

Comment:
Wszystko zadziała jesli bedziesz uzywac tylko GET/POST, ale jezeli mamy opcje uzywania innych metod HTTP to dlaczego mamy tego nie robić? Pozniej jak piszesz np API dla frontendu i zrobisz taki myk ze chcesz usunac obiekt a zrobisz GET to ktos kto uzywa tego API moze pomyslec ze jestes strasznym leszczem ze nie znasz podstaw ;)